main.py

```diff
@@ -296,35 +320,22 @@ def evaluate_genomes(genomes, config):
                     keys[key] = key in ai_chosen_moves
 
                 # ! If any moves are True, check for conflicting moves
-                conflicting_moves = False
-                if any(keys.values()):
-                    # TODO: Better define later
-                    conflicting_moves = (
-                        keys[pygame.K_LEFT] and keys[pygame.K_RIGHT]
-                    ) or (keys[pygame.K_DOWN] and keys[pygame.K_UP])
-                player.conflicting_moves = conflicting_moves
+                if any(keys):
+                    correct_conflicting_movements(keys)
                 player.in_motion = False

```

Comment:
Can't tell if these are correctly indented? Player move time shouldn't be set if the player hasn't moved. No need to needlessly call functions too if the keys are all false
![image](https://github.com/a-curious-coder/sentient_blobs/assets/45906280/768c7f27-901a-472d-aaed-5d75ae01bb21)

Reply:
done